micro-benchmarks/DRB172-parallel-orig-no.c

```diff
@@ -0,0 +1,68 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+/**
+ * BT.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 bt suit.
+ * Tsan and Intel Inspector can not correctly analysis the nested parallel for, and report a false positive.
+*/
+
+#include <stdio.h>
+
+
+int main(int argh, char* argv[]){
+  int i,j,k,m;
+  int nthreads = 1;
+  double u[12][12][12][5];
+  double tmp1,tmp2,tmp3;
+  double fjac[12][12][12][5][5];
+
+  /*initial m */
+  #pragma omp for
```

Comment:
Another bad OpenMP usage: "omp for" must be used inside "omp parallel" to be meaningful.